script/ci-build

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -v
+
+export DIR=`pwd`
+
+if [ "x$RUBY_SASS" = "xyes" ]; then
+  for GEMFILE in `ls $DIR/Gemfile*`
```

Comment:
The following lines are better replaced with `find -name ... \! -name ... -print | while read GEMFILE; do` shell idiom